backend/src/main/java/zipgo/admin/dto/PetFoodUpdateRequest.java

```diff
@@ -0,0 +1,30 @@
+package zipgo.admin.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.List;
+
+public record PetFoodUpdateRequest (
+        @JsonProperty("pet_food_name") String petFoodName,
+        @JsonProperty("brand_name") String brandName,
+        @JsonProperty("eu_standard") boolean euStandard,
+        @JsonProperty("us_standard") boolean usStandard,
+        @JsonProperty("image_url") String imageUrl,
+        @JsonProperty("functionalities") List<String> functionalities,
+        @JsonProperty("primary_ingredients") List<String> primaryIngredients
```

Comment:
@JsonNaming을 쓰는건 어떨까요?

[요렇게 뺄 수 있을 것 같아요~](https://github.com/woowacourse-teams/2023-zipgo/blob/develop/backend/src/main/java/zipgo/auth/infra/kakao/dto/KakaoTokenResponse.java)

```suggestion
@JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy.class)
public record PetFoodUpdateRequest (
        String petFoodName,
        String brandName,
        boolean euStandard,
        boolean usStandard,
        String imageUrl,
        List<String> functionalities,
        List<String> primaryIngredients
```

Reply:
지렸다